pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -657,7 +657,7 @@ type BackupStorageS3Spec struct {
 	Prefix                string `json:"prefix,omitempty"`
 	Region                string `json:"region,omitempty"`
 	EndpointURL           string `json:"endpointUrl,omitempty"`
-	CredentialsSecret     string `json:"credentialsSecret"`
+	CredentialsSecret     string `json:"credentialsSecret,omitempty"`
```

Comment:
it doesn't look right to me? shouldn't it be required?

Reply:
Good question) I would like to discuss it in PR) We have task https://jira.percona.com/browse/K8SPSMDB-185
It means that we can skip this field in eks if we have necessary settings. This option doesn't work now because of crd validation. What is the best way to fix it?

Reply:
aah I see